PubNub/Data/Service Objects/PNSubscribeStatus.h

```diff
@@ -31,12 +31,14 @@
 @property (nonatomic, readonly, strong) NSString *actualChannel;
 
 /**
- @brief  Time at which even arrived.
+ @brief  Time at which event arrived.
  
  @since 4.0
  */
 @property (nonatomic, readonly, strong) NSNumber *timetoken;
 
+@property (nonatomic, readonly, strong) NSNumber *region;
```

Comment:
@jzucker2 Is there any real reasons to expose region information to the user? If some debug information at the end should be presented to the user, I suggest to wrap it into separate class (like in 3.x version) and store resulting instance. Access to such instance should be described in _PNResult_ as base class which is base class for all objects.